documentation/assemblies/configuring/assembly-storage.adoc

```diff
@@ -13,11 +13,15 @@ The supported storage types are:
 * Ephemeral (Recommended for development only)
 * Persistent
 * JBOD (Kafka only; not available for ZooKeeper)
+* Tiered storage (Technology preview)
 
-To configure storage, you specify `storage` properties in the custom resource of the component. 
+To configure non-tiered storage, you specify `storage` properties in the custom resource of the component. 
```

Comment:
I would not use `non-tiered`. This is essentially one tier of the tiered storage. I would keep this as it was. If you want to more differentiate it, you can maybe elaborate on what tiered storage is on the line 21 or after it.

Reply:
Right. I've removed that and added a line on tiered storage: 

> Tiered storage provides more flexibility for data management by leveraging the use of both fast local storage and slower, more cost-effective remote storage solutions.